examples/amqp_listen.c

```diff
@@ -133,6 +133,8 @@ int main(int argc, char const *const *argv) {
     }
   }
 
+  free(queuename.bytes);
```

Comment:
Use [`amqp_bytes_free`](https://github.com/alanxz/rabbitmq-c/blob/master/librabbitmq/amqp.h#L1008-L1023) to release memory. E.g., `amqp_bytes_free(queuename)`